CONTRIBUTING.rst

```diff
@@ -9,12 +9,30 @@ get an idea of how to connect and participate in pycsw community and development
 Contributions and Licensing
 ---------------------------
 
-Contributors are asked to provide a Contributor License Agreement for working on the project:
+Contributors are asked to confirm that they comply with project `license <https://github.com/geopython/pycsw/blob/master/LICENSE.txt>`_ guidelines.
 
-- `Corporate`_
-- `Individual`_
+GitHub Commit Access
+^^^^^^^^^^^^^^^^^^^^
+
+- proposals to provide developers with GitHub commit access shall be emailed to the pycsw-devel `mailing list`_.  Proposals shall be approved by the pycsw development team.  Committers shall be added by the project admin
+- removal of commit access shall be handled in the same manner
+- each committer must send an email to the pycsw mailing list agreeing to the license guidelines (see `Contributions and Licensing Agreement Template <#contributions-and-licensing-agreement-template>`_)
+- each committer shall be listed in https://github.com/geopython/pycsw/blob/master/COMMITTERS.txt
+
+Pull Requests
+^^^^^^^^^^^^^
+
+- pull requests can provide agreement to license guidelines as text in the pull request or via email to the pycsw `mailing list`_  (see `Contributions and Licensing Agreement Template <#contributions-and-licensing-agreement-template>`_)
```

Comment:
@amercader @capooti thanks for the comments.  Updated to clarify text in https://github.com/tomkralidis/pycsw/commit/f1756b312d204175ecf36939ce8ba6af1e0466ed